packages/gatsby-remark-images/src/gatsby-node.js

```diff
@@ -1,75 +1,73 @@
 
       Note:
       - set this option to transparent for a transparent image background.
       - set this option to none to completely remove the image background.`),
-      quality: Joi.number()
-        .default(50)
-        .description(`The quality level of the generated files.`),
-      withWebp: Joi.boolean()
-        .default(false)
-        .description(
-          `Additionally generate WebP versions alongside your chosen file format. They are added as a srcset with the appropriate mimetype and will be loaded in browsers that support the format. Pass true for default support, or an object of options to specifically override those for the WebP files. For example, pass { quality: 80 } to have the WebP images be at quality level 80.`
-        ),
-      tracedSVG: Joi.boolean()
-        .default(false)
-        .description(
-          `Use traced SVGs for placeholder images instead of the “blur up” effect. Pass true for traced SVGs with the default settings (seen here), or an object of options to override the default. For example, pass { color: "#F00", turnPolicy: "TURNPOLICY_MAJORITY" } to change the color of the trace to red and the turn policy to TURNPOLICY_MAJORITY. See node-potrace parameter documentation for a full listing and explanation of the available options.`
-        ),
-      loading: Joi.string()
-        .valid(`lazy`, `eager`, `auto`)
-        .default(`lazy`)
-        .description(
-          `Set the browser’s native lazy loading attribute. One of lazy, eager or auto.`
-        ),
-      disableBgImageOnAlpha: Joi.boolean()
-        .default(false)
-        .description(
-          `Images containing transparent pixels around the edges results in images with blurry edges. As a result, these images do not work well with the “blur up” technique used in this plugin. As a workaround to disable background images with blurry edges on images containing transparent pixels, enable this setting.`
-        ),
-      disableBgImage: Joi.boolean()
-        .default(false)
-        .description(
-          `Remove background image and its’ inline style. Useful to prevent Stylesheet too long error on AMP.`
-        ),
-      srcSetBreakpoints: Joi.array()
-        .items(Joi.number())
-        .description(
-          `By default gatsby generates 0.25x, 0.5x, 1x, 1.5x, 2x, and 3x sizes of thumbnails. If you want more control over which sizes are output you can use the srcSetBreakpoints parameter. For example, if you want images that are 200, 340, 520, and 890 wide you can add srcSetBreakpoints: [ 200, 340, 520, 890 ] as a parameter. You will also get maxWidth as a breakpoint (which is 650 by default), so you will actually get [ 200, 340, 520, 650, 890 ] as breakpoints.`
-        ),
-    })
-  }
+    quality: Joi.number()
+      .default(50)
+      .description(`The quality level of the generated files.`),
+    withWebp: Joi.boolean()
```

Comment:
It looks like we can no longer pass an object with quality config for `withWebp` option?

Reply:
Woops, it looks something that we missed here. I'll make it back :)

Reply:
Available in https://github.com/gatsbyjs/gatsby/pull/27944 😊